src/Deck.js

```diff
@@ -0,0 +1,11 @@
+class Deck {
+  constructor(cards) {
+    this.cards = cards;
+  }
```

Comment:
Passing in all the cards as a single array of objects will make manipulating the deck much easier in future iterations.